### PR TITLE
Support Tailwind

### LINKED ISSUE #5: Tailwind
Make UI sexy :scream:

### LINKED ISSUE #6: Extend `flake.nix` to support Node + Tailwind

### LINKED ISSUE #7: Update CI to install `Tailwind` and its dependencies

### PR DESCRIPTION
**Pull request by sectore**
- [x] Support Tailwind
- [x] Add fonts, basic styles tec. - all based on [Fedimints `ui` ](https://github.com/fedimint/ui/)
- [x] Style `Join` screen
- [x] Style `Send` screen
- [x] Style `Receive` screen
- [x] Extend `flake.nix` to support Node + Tailwind
- [x] Update CI to install npm dependencies (needed by Tailwind) c15f82b

## Demo

[Screencast from 17.08.2023 14:22:30.webm](https://github.com/elsirion/fedimint-leptos-test/assets/47693/fe54a260-73bf-429f-979f-11581d64ec00)

Closes #5 
Closes #6 
Closes #7

### CODE REVIEW

File: src/components/logo.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use leptos::*;
+
+#[component]
+pub fn Logo(cx: Scope) -> impl IntoView {
+    // Fedimint Logo - Figma
+    // https://www.figma.com/file/ioeAVbWBc3ow2RaZETatM7/Fedimint?type=design&node-id=5520-65113&mode=design&t=T7dxEPEa81LH5HYg-0
+    view! { cx,
```
Comment (elsirion) [inline src/components/logo.rs:7]: Since this is not an official Fedimint project I'd avoid using the logo so prominently.

File: README.md [hunk 2]
```diff
@@ -5,12 +5,23 @@
 This repo contains a proof-of-concept of how to integrate `fedimint-client` with the Leptos web framework to build a
 PWA. There are still some bugs being worked out, but there is a demo available at https://fedimint-leptos.sirion.io/.
 
-## (Temporary) Pre-requirements (will be changed for sure)
+## Pre-requirements
```
Comment (elsirion) [inline README.md:8]: I feel like "Prerequisites" would be the more common term

**Review by elsirion — COMMENTED** [review]
Really cool! Having a way to style the app will allow us to engage the design community. I'd like to get to a point where this project has its own visual identity, but for now using the setup UI design is a good first step. I also want to get some ideas for naming … I kinda like "webimint" :laughing:  but I know I'm horrible at naming things so will get some feedback on discord.

**Comment by elsirion** [comment]
I started a thread about naming: https://discord.com/channels/990354215060795454/990354215878688860/threads/1140921151707684984

File: src/components/logo.rs (hunk 1 above)
Comment (sectore) [reply src/components/logo.rs:7]: Ok, will move it into a footer.

File: README.md (hunk 2 above)
Comment (sectore) [reply README.md]: Done 53b982a3955bc8ec7d4eddad97a3a91accc2d761

**Comment by sectore** [comment]
> /home/runner/work/_temp/367c47d9-5002-4ea2-8a15-855932237d5f.sh: line 1: trunk: command not found

^ https://github.com/elsirion/fedimint-leptos-test/actions/runs/5890859971/job/15976808584?pr=8#step:6:914

@elsirion Any idea why `trunk` is missing on CI?

File: README.md [hunk 3]
```diff
@@ -44,7 +67,9 @@ fedimint-leptos-test$ trunk serve
 2023-08-06T12:00:42.919899Z  INFO 📡 server listening at http://127.0.0.1:8080
 ```
 
-Enter invite code into to input and submit. Check `Pre-requirements` above to see how to get it.
+2. Open [http://127.0.0.1:8080](http://127.0.0.1:8080) in your browser
+
+3. Enter invite code into to input and submit. Check `Pre-requirements` above to see how to get it.
```
Comment (elsirion) [inline README.md:72]: nit: Missing rename to "prerequisites", could also be a link instead of inline code.

**Review by elsirion — APPROVED** [review]
I mostly tested that everything works and didn't do a deep dive on the code yet, but high level it looks very clean :smiley:

**Pull request merged**